src/Symfony/Component/Cache/Traits/AbstractTrait.php

```diff
@@ -268,7 +268,7 @@ private function getId($key): string
                     $this->namespaceVersion = $v;
                 }
                 if ('1'.static::NS_SEPARATOR === $this->namespaceVersion) {
-                    $this->namespaceVersion = substr_replace(base64_encode(pack('V', time())), static::NS_SEPARATOR, 5);
+                    $this->namespaceVersion = self::createNewNamespaceVersion();
```

Comment:
we really want `time()` here, so that concurrent front servers generate the same version statistically when the first version number is created